src/main/java/org/fundacionjala/coding/oscar/DnaStrand.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.coding.oscar;
+
+/**
+ * Created by Oz64 on 14/09/2017.
+ */
+public final class DnaStrand {
+    /**
+     * constructor.
+     */
+    private DnaStrand() {
+    }
+
+    /**
+     * This method change the letter for them complement.
+     *
+     * @param stringDNA is the DNA string.
+     * @return the new string.
+     */
+    public static String makeComplement(String stringDNA) {
+        String[] list = stringDNA.split("");
+        for (int index = 0; index < list.length; index++) {
+            switch (list[index]) {
```

Comment:
For this scenario.
Don't use switch case or if else
Try to use String.replace method.